src/content/reference/okteto-cli.mdx

```diff
@@ -790,6 +790,26 @@ Displays the current installed version
 $ okteto version
 ```
 
+### test
+
+Runs tests using Okteto infrastructure via [Remote Execution](core/remote-execution.mdx#how-remote-execution-works). `okteto test` builds and deploys your application if it hasn't yet been deployed. If the application is deployed it is not redeployed unless the `--deploy` flag is used.
+All application endpoints are available inside the test exection via the namespaced host. For example, if you deployed an api service on port 8080 it will be accesible at: `http://api.${OKTETO_NAMESPACE}:8080`.
+
+[Dynamic endpoints](core/okteto-variables.mdx#configuring-dynamic-endpoints) are available within test section.
+
+
+| Options       |   Type   | Description                                                                                                            | Default     |
+| :------------ | :------: | :--------------------------------------------------------------------------------------------------------------------- | :---------- |
+| _--file_      | (string) | The path to the manifest file                                                                                          | "okteto.yaml" |
+| _--namespace_ | (string) | The namespace to use                                                                                                   | Current Okteto Namespace |
+| _--name_      | (string) | The name of the development environment name to be deployed                                                            | |
+| _--context_   | (string) | The context to use                                                                                                     | Current Okteto Context |
+| _--deploy_    |  (bool)  | Force execution of the commands in the `deploy` section of the okteto manifest                                         | `false` |
+| _--no-cache_  |  (bool)  | Do not use the cache for running the tests                                                                             | `false` |
+| _--var_       |  (list)  | Set a pipeline variable (can be set more than once)                                                                    | |
```

Comment:
@maroshii we do support `--var` but we don't support `variables` in the test container definition, right?

Reply:
Yes, I think that we only support the flag, right? Just asking in case we are missing `variables`, but if it's not supported, it's fine for now. `--var` should say something like `Set an environment variable for the test container (can be set more than once)`